pkgs/top-level/all-packages.nix

```
@@ -41394,7 +41394,9 @@ with pkgs;
 
   wacomtablet = libsForQt5.callPackage ../tools/misc/wacomtablet { };
 
-  wamr = callPackage ../development/interpreters/wamr { };
+  wamr = callPackage ../development/interpreters/wamr {
+    stdenv = if stdenv.isDarwin then darwin.apple_sdk_11_0.stdenv else stdenv;
+  };
```

```suggestion
  wamr = darwin.apple_sdk_11_0.callPackage ../development/interpreters/wamr { };
```
this defaults to stdenv on linux and uses the correct sdks

---

Nice, thank you. Updated.